CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.25.1 / 2021-03-13
+
+* [BUGFIX] Fix a crash in `promtool` when a subquery with default resolution is used. #8569
+* [BUGFIX] Fix a bug that could return duplicate of datapoints in /api/v1/query. #8591
```

Comment:
Nit: Remove the "of". Or even more compact: "Fix possible return of duplicate datapoints in /api/v1/query."

Reply:
thanks, applied.